aurora/core.py

```diff
@@ -265,7 +268,7 @@ def setup_kin_profs_depts(self):
         self.total_source = np.pi*np.sum(self.src_core*S0*(self.rvol_grid/self.pro_grid)[:,None],0)  # sum over radius
         self.total_source += self.src_div   # units of particles/s/cm
 
-        if self.wall_recycling>0: # recycling activated
+        if self.wall_recycling >= 0: # recycling activated
```

Comment:
I don't think that this is right. Recycling is only really activated when `wall_recycling>0`. If `wall_recycling=0`, we have divertor "return fluxes", but no real recycling. That's the strange nomenclature/convention that we currently have, sort of following the one in STRAHL. We could change this too, but it likely wouldn't be backwards-compatible...

Reply:
The edge transport model is turned off when recycling_flag is False. In this case, the self.wall_recycling in aurora is set to -1. When self.wall_recycling is 0, the model should be still turned on, only no particles trapped on the wall return back in the plasma. As it was done before, for R = 0.0001 and R = 0 the results were completely different.